sxg_rs/src/headers.rs

```diff
@@ -442,6 +455,21 @@ pub fn parse_accept_level(accept: &str) -> AcceptLevel {
     }
 }
 
+fn parse_chrome_major_version(user_agent: &str) -> Option<u32> {
+    parse_number_after(user_agent, " Chrome/")
+        .or_else(|| parse_number_after(user_agent, " Chromium/"))
+}
+
+// Find the substring coming after `pattern`, and parse it as a number.
+fn parse_number_after(input: &str, pattern: &str) -> Option<u32> {
+    let i = input.find(pattern)? + pattern.len();
+    if i >= input.len() {
+        return None;
+    }
+    let (x, _) = input[i..].split_once('.')?;
```

Comment:
LGTM. I was worried about cases where there is only one number after the slash, but I couldn't find any instances.